environment/interpretation/lane_detector/launch/module.launch

```diff
@@ -3,17 +3,17 @@
 <launch>
     
     <node pkg="lane_detector" type="lane_detector" name="lane_detector1" args="lane_detector1" output="screen" cwd="node">
-        <param name="debug_mode" type="int" value="0"/>
-        <param name="ipt_offsets_file" type="str" value="../data/ipt_offsets1.txt"/>                
+        <param name="debug_mode" type="int" value="8"/>
+        <param name="ipt_offsets_file" type="str" value="ipt_offsets1.txt"/>                
         <param name="map_size" type="int" value="1000"/>        
         <param name="published_topic_name" type="str" value="/lane_detector1/lanes"/>        
         <param name="subscribed_topic_name" type="str" value="/logitech_camera1/image"/>
-        <param name="training_data_file" type="str" value="../data/Test4"/>
+        <param name="training_data_file" type="str" value="Samples1"/>
```

Comment:
@abinashmeher999 I have already used ros::package::getPath() in code to get the path of directory itself. Hence, it shouldn't act as problem. Same goes for other